README.md

```diff
@@ -35,9 +39,13 @@ The configuration is in JSON. An example with all possible options:
   ]
 }
 ```
+
 Name     | Description
 ---------|------------
 hostPort | The host and port to connect to via remote JMX. If not specified, will talk to the local JVM.
+hostUrl | The full JMX URL.  If not specified, will use hostPort.
```

Comment:
We should probably split out a separate "Connection" section, as this is a bit hard to understand now.